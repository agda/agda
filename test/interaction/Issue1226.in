ioTCM currentFile None Direct (cmd_load currentFile [])
ioTCM currentFile Interactive Direct (cmd_refine 1 noRange "unit")
ioTCM currentFile Interactive Direct (cmd_autoOne 0 noRange "")
