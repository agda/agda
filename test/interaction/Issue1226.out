(agda2-status-action "")
(agda2-info-action "*Type-checking*" "" nil)
(agda2-highlight-clear)
(agda2-status-action "")
(agda2-info-action "*All Goals*" "?0 : Unit ?1 : Unit " nil)
((last . 1) . (agda2-goals-action '(0 1)))
(agda2-give-action 1 "unit")
(agda2-status-action "")
(agda2-info-action "*All Goals*" "?0 : Unit " nil)
((last . 1) . (agda2-goals-action '(0)))
(agda2-give-action 0 "unit")
((last . 1) . (agda2-goals-action '()))
